srcpkgs/spek-x/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'spek-x'
+pkgname=spek-x
+version=0.9.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel intltool libtool pkg-config
+ wxWidgets-common wxWidgets-gtk3-devel"
+makedepends="ffmpeg6-devel wxWidgets-gtk3-devel"
+depends="ffmpeg6"
+short_desc="Acoustic spectrum analyser"
+maintainer="zlice <zlice555@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/MikeWang000000/spek-x"
+distfiles="https://github.com/MikeWang000000/spek-x/archive/v${version}.tar.gz"
+checksum="7f4b109903333cdd310ab2b9176f0dc49a565891376e281a233d342cedf90ef7"
+
+CXXFLAGS="-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64"
+
+# Tell autogen.sh to use "wx-config-gtk3" instead of "wx-config"
+export WX_CONFIG_NAME="wx-config-gtk3"
+
+pre_configure() {
+	NOCONFIGURE=1 ./autogen.sh
+}
+
+spek-alternative_package() {
```

Comment:
missing subpackage symlink

Reply:
![spek1](https://github.com/user-attachments/assets/07a938aa-5551-46dd-8f8b-3c7b7c4004bb)

![spek2](https://github.com/user-attachments/assets/8bd6b536-e6e8-49c5-a7d1-b4b96425f872)


not sure what you mean, different than this?

Reply:
ah missed it